backend/secrets.py

```diff
@@ -2,3 +2,6 @@
 # https://console.firebase.google.com/project/trogdors-29fa4/settings/serviceaccounts/databasesecrets
 FIREBASE_SECRET = 'd0h365xNGuJU7A3rB65bAJ4v59VMLJiaNY5R0D6C'
 FIREBASE_EMAIL = ''
+IONIC_TOKEN = ''
+IONIC_APPID = ''
+IONIC_SECURITY_TAG = ''
```

Comment:
Yes. I still need to talk to you on how these values get set when you deploy to prod.